ngsa/src/main/java/com/cse/ngsa/app/utils/CommonUtils.java

```diff
@@ -156,4 +160,21 @@ public static boolean isNullWhiteSpace(final String string) {
     return string == null || string.isEmpty() || string.trim().isEmpty();
   }
 
+  /**
+   * Prepares burst header values.
+   */
+  public String getBurstHeaderValue() {
+    long cpuLoad = Math.round(ManagementFactory.getPlatformMXBean(
+        com.sun.management.OperatingSystemMXBean.class).getProcessCpuLoad() * 100L);
+
+    String burstHeaderValue = String.format(
+        "service=%s, current-load=%s, target-load=%s, max-load=%s ",
+        environment.getProperty("service.name"),
```

Comment:
this would be a good default value if `--burst-service` isn't passed, though it would still be nice to have that option.